en/config_mc/racer_setup.md

```diff
@@ -112,93 +114,40 @@ Make sure to calibrate the ESCs before doing any tuning.
-- Visible random small twitches.
-
-The best filter settings depend on the vehicle.
-The defaults are set conservatively — such that they work on lower-quality setups as well.
-
-#### Filter Tuning
-
-First make sure to have the high-rate logging profile activated ([SDLOG_PROFILE](../advanced_config/parameter_reference.md#SDLOG_PROFILE) parameter).
-[Flight Review](../getting_started/flight_reporting.md) will then show an FFT plot for the roll, pitch and yaw controls.
-
-:::warning
-Do not try to fix a vehicle that suffers from high vibrations with filter tuning.
-Instead fix the vehicle hardware setup.
-:::
-
-Filter tuning is best done by reviewing flight logs.
-You can do multiple flights right after each other with different parameters and then inspect all logs, but make sure to disarm in between so that separate log files are created.
-
-The performed flight maneuver can simply be hovering in [Manual/Stabilized mode](../flight_modes/manual_stabilized_mc.md) with some rolling and pitching to all directions and some increased throttle periods.
-The total duration does not need to be more than 30 seconds.
-In order to better compare, the maneuver should be similar in all tests.
-
-First tune the gyro filter [IMU_GYRO_CUTOFF](../advanced_config/parameter_reference.md#IMU_GYRO_CUTOFF) by increasing it in steps of 10 Hz while using a low D-term filter value ([IMU_DGYRO_CUTOFF](../advanced_config/parameter_reference.md#IMU_DGYRO_CUTOFF) = 30).
-Upload the logs to https://logs.px4.io and compare the *Actuator Controls FFT* plot.
-Set the cutoff frequency to a value before the noise starts to increase noticeably (for frequencies around and above 60 Hz).
-Then tune the D-term filter (`IMU_DGYRO_CUTOFF`) in the same way.
+### Filter Tuning
 
-Below is an example for three different filter values (40Hz, 70Hz, 90Hz).
-At 90 Hz the general noise level starts to increase (especially for roll), and thus a cutoff frequency of 70 Hz is a safe setting.
-![IMU_DGYRO_CUTOFF=40](../../assets/airframes/multicopter/racer_setup/actuator_controls_fft_dgyrocutoff_40.png)
-![IMU_DGYRO_CUTOFF=70](../../assets/airframes/multicopter/racer_setup/actuator_controls_fft_dgyrocutoff_70.png)
-![IMU_DGYRO_CUTOFF=90](../../assets/airframes/multicopter/racer_setup/actuator_controls_fft_dgyrocutoff_90.png)
-
-:::note
-The plot cannot be compared between different vehicles, as the y axis scale can be different.
-On the same vehicle it is consistent and independent of the flight duration though.
-:::
+Filters trade off control latency and noise filtering, both of which impact performance.
+For information see: [Filter/Control Latency Tuning](../advanced_config/filter_tuning.md)
 
 ### PID Tuning (Second Round)
 
-Now do a second round of PID tuning, this time as tight as possible, and also tune the thrust curve.
+Now do a second round of PID tuning, this time as tight as possible, and also tuning the thrust curve.
```

Comment:
@bkueng Note, this now points to the advanced tuning guide for the thrust curve.